### PR TITLE
Print error messages to stderr in quickstart example

### PR DESCRIPTION
**Pull request by test9753**
Use `stderr` to print error messages.

### CODE REVIEW

File: examples/quickstart.rs [hunk 1]
```diff
@@ -19,23 +19,26 @@ mod errors {
...
-            println!("backtrace: {:?}", backtrace);
-        }
-
-        ::std::process::exit(1);
-    }
-}
+fn main() {                                                                         
+    if let Err(ref e) = run() {                                                     
+        use ::std::io::Write;                                                       
+        let (stderr, errmsg) = (&mut ::std::io::stderr(), "Error writing to stderr");
```
Comment (Yamakaky) [inline examples/quickstart.rs]: Use two `let` declarations, pattern matching doesn't add any value in this case.
Also, I'm not sure you need the `&mut` part.

**Comment by Yamakaky** [comment]
Good idea! Could you also update `quick_main`?

**Comment by test9753** [comment]
CI build errors (with nightly) seems to be unrelated to this PR

**Comment by Yamakaky** [comment]
Thanks!

**Pull request merged**